package.xml

```diff
@@ -22,14 +22,17 @@
 
   <exec_depend>gazebo_ros</exec_depend>
   <exec_depend>opencv3</exec_depend>
+  <exec_depend>tinyxml</exec_depend>
```

Comment:
Why not in testing, the cmake_modules and tinyxml?

Reply:
test_depend is for dependency that exclusively for testing...

To be honest, I don't really know where you are using tinyxml here